### PR TITLE
Examples: Minor boilerplate reduction

### PR DESCRIPTION
**Pull request by DLehenbauer**
No framework change, just updating some copy/pasted sample code to be a bit less verbose.

(Was experimenting with a different change.  This minor boilerplate reduction fell out.  Thought why not keep it?)

### CODE REVIEW

File: examples/data-objects/codemirror/src/codemirror.ts [hunk 1]
```diff
@@ -266,20 +266,16 @@ class SmdeFactory implements IFluidDataStoreFactory {
...
         const runtimeClass = mixinRequestHandler(
             async (request: IRequest) => {
                 const router = await routerP;
                 return router.request(request);
             });
 
-        const runtime = new runtimeClass(context, dataTypes);
+        const runtime = new runtimeClass(context, new Map([
+            SharedMap.getFactory(),
+            SharedString.getFactory(),
```
Comment (DLehenbauer) [inline examples/data-objects/codemirror/src/codemirror.ts:277]: Note: 'sequenceFactory' was a misleading name.  Original was actually a SharedString.

File: examples/data-objects/prosemirror/src/prosemirror.ts [hunk 2]
```diff
@@ -176,20 +176,16 @@ class ProseMirrorFactory implements IFluidDataStoreFactory {
...
         const runtimeClass = mixinRequestHandler(
             async (request: IRequest) => {
                 const router = await routerP;
                 return router.request(request);
             });
 
-        const runtime = new runtimeClass(context, dataTypes);
+        const runtime = new runtimeClass(context, new Map([
+            SharedMap.getFactory(),
+            SharedString.getFactory(),
```
Comment (DLehenbauer) [inline examples/data-objects/prosemirror/src/prosemirror.ts:187]: Ditto: 'sequenceFactory' was a SharedString.

**Review by curtisman — APPROVED** [review]
:shipit:

**Comment by DLehenbauer** [comment]
Thanks, @curtisman !

**Pull request merged**